### ISSUE TITLE
[EPIC] Dependencies bump for v1.1.0

### ISSUE THREAD

**Issue by bk201** [harvester/harvester#2927]
We need to bump these dependencies for v1.1.0-rc3
- [x] Rancher v2.6.9. 
- [x] RKE2 v1.24.7+rke2r1
- [x] Longhorn v1.3.2

**Comment by bk201** [comment]
Test plan:
Verify a new deployment has these versions:
- Rancher v2.6.9. (`kubectl get settings.management.cattle.io server-version`)
- RKE2 v1.24.7+rke2r1 (`kubectl get nodes` -> `VERSION`)
- Longhorn v1.3.2 (`kubectl get ds longhorn-manager -n longhorn-system -o yaml | grep image` -> should see `longhornio/longhorn-manager:v1.3.2`). Or via Longhorn GUI to see the version.

**Comment by TachunLin** [comment]
Verified **fixed** on `master-7b698413-head` (10/12). Close this issue.

### Result
 Verified on the newly install 3 nodes Harvester cluster:
 
* Checked the Rancher version: v2.6.9
  ```
  harvester-node-0:~ # kubectl get settings.management.cattle.io server-version
  NAME             VALUE
  server-version   v2.6.9
  ```

* Checked the RKE2 version: v1.24.7+rke2r1
  ```
  harvester-node-0:~ # kubectl get nodes
  NAME               STATUS   ROLES                       AGE   VERSION
  harvester-node-0   Ready    control-plane,etcd,master   86m   v1.24.7+rke2r1
  harvester-node-1   Ready    control-plane,etcd,master   71m   v1.24.7+rke2r1
  harvester-node-2   Ready    control-plane,etcd,master   60m   v1.24.7+rke2r1
  ```

* Checked the Longhorn version: v1.3.2
   ```
   harvester-node-0:~ # kubectl get ds longhorn-manager -n longhorn-system -o yaml | grep image
          - --engine-image
          - --instance-manager-image
          - --share-manager-image
          - --backing-image-manager-image
          - longhornio/backing-image-manager:v3_20221003
          - --manager-image
          image: longhornio/longhorn-manager:v1.3.2
          imagePullPolicy: IfNotPresent
          image: longhornio/longhorn-manager:v1.3.2
          imagePullPolicy: IfNotPresent
   ```

### Test Information
* Test Environment: 3 nodes harvester on local kvm machine
* Harvester version: `master-7b698413-head` (10/12)

### Verify Steps
1. Prepare a 3 nodes Harvester cluster 
2. Check Rancher is v2.6.9. (`kubectl get settings.management.cattle.io server-version`)
3. Check RKE2 is v1.24.7+rke2r1 (`kubectl get nodes` -> VERSION)
4. Check Longhorn v1.3.2 (`kubectl get ds longhorn-manager -n longhorn-system -o yaml | grep image` -> should see `longhornio/longhorn-manager:v1.3.2`)
